frontend/packages/data-portal/app/components/Deposition/DepositionMetadataTable.tsx

```diff
@@ -0,0 +1,129 @@
+import { Icon } from '@czi-sds/components'
+import { useState } from 'react'
+
+import { AccordionMetadataTable } from 'app/components/AccordionMetadataTable'
+import { AuthorLegend } from 'app/components/AuthorLegend'
+import { AuthorInfo } from 'app/components/AuthorLink'
+import { AuthorList } from 'app/components/AuthorList'
+import { DatabaseEntryList } from 'app/components/DatabaseEntry'
+import { Deposition } from 'app/hooks/useDepositionById'
+import { useI18n } from 'app/hooks/useI18n'
+import { getTableData } from 'app/utils/table'
+
+function CollapsibleDescription({ text }: { text: string }) {
+  const { t } = useI18n()
+  const [collapsed, setCollapsed] = useState(true)
+
+  return (
+    <div>
+      <p className={collapsed ? 'text-ellipsis line-clamp-3' : undefined}>
+        {text}
+      </p>
+      <div className="mt-sds-s font-semibold text-sds-primary-400">
+        <button type="button" onClick={() => setCollapsed(!collapsed)}>
+          <span className="flex flex-row gap-sds-xxs items-center">
+            {collapsed ? (
+              <>
+                <Icon
+                  sdsIcon="plus"
+                  sdsSize="xs"
+                  sdsType="static"
+                  className="!text-current"
+                />
+                {t('showMore')}
+              </>
+            ) : (
+              <>
+                <Icon
+                  sdsIcon="minus"
+                  sdsSize="xs"
+                  sdsType="static"
+                  className="!text-current"
+                />
+                {t('showLess')}
+              </>
+            )}
+          </span>
+        </button>
+      </div>
+    </div>
+  )
+}
+
+export function DepositionMetadataTable({
+  deposition,
+  initialOpen,
+}: {
+  deposition: Deposition
+  initialOpen?: boolean
```

Comment:
`isInitialOpen`

Reply:
i dont quite like this since using its shorthand would look weird in a component since it sounds like a question: `<DepositionMetadataTable deposition={deposition} isInitialOpen />`

maybe something like `openOnInit` or `openInitially`?